web/models.py

```
@@ -2805,3 +2805,169 @@
+    end_time = models.DateTimeField()
+    location = models.CharField(max_length=200, blank=True, help_text="Physical location of the event")
+    meeting_link = models.URLField(blank=True, help_text="Virtual meeting link")
+    image = models.ImageField(upload_to='chapter_events', blank=True)
+    max_participants = models.PositiveIntegerField(default=50)
+    is_public = models.BooleanField(default=True, help_text="If false, only chapter members can view")
+    organizer = models.ForeignKey(User, on_delete=models.CASCADE, related_name='organized_chapter_events')
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+
+    class Meta:
+        ordering = ['-start_time']
```

Ruff (0.8.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)